css/src/atomics/width.scss

```diff
@@ -42,3 +42,13 @@ $all-widths: list.join($universal-widths, $mobile-incompatible-widths);
 		}
 	}
 }
+
+// max-widths
+
+.max-width-full {
+	max-width: 100% !important;
+}
+
+.max-width-100vw {
```

Comment:
This one is a sticking point for me, as I think it may cause inadvertent side scroll when used (because vw does not account for scrollbars).